Package.swift

```diff
@@ -5,9 +5,9 @@ let package = Package(
     name: "jwt-kit",
     platforms: [
         .macOS(.v13),
```

Comment:
Do we want to bump this to 12 to line up?

Reply:
There's a few errors around EC keys about missing conformances with `.macOS` < 13. Also `JSONDecoder` and `JSONEncoder` are only `Sendable` in 13+

Reply:
I mean we can try solving them but not sure it's worth it?

Reply:
What are the issues with EC keys? Because surely we'll have the same issues on iOS 15

Reply:
I wonder if it's worth being _really_ annoying and adding CI for iOS (including 15) so see what issues we'd hit

Reply:
The thing is Crypto doesn't specify iOS requirements (or any besides macOS) so there's no issues with that